src/options.c

```diff
@@ -335,6 +335,10 @@ void get_cmdln_options(int argc, char *argv[]) {
         pwd_entry=getpwuid(getuid());
         if (pwd_entry!=NULL) {
             str=(char*)malloc(strlen(pwd_entry->pw_dir)+14);
+            if(!str) {
+              printf("Fatal: failed to allocate %zu bytes.\n", strlen(pwd_entry->pw_dir)+14);
```

Comment:
Help the user to identify the problem
```suggestion
              printf("Fatal: failed to allocate %zu bytes reading user directory for config file.\n", strlen(pwd_entry->pw_dir)+14);
```